dissect/hypervisor/disk/vmdk.py

```diff
@@ -420,11 +451,17 @@ def parse(cls, vmdk_config):
                 continue
 
             if line.startswith("RW ") or line.startswith("RDONLY ") or line.startswith("NOACCESS "):
-                access_type, size, extent_type, filename = line.split(" ", 3)
-                filename = filename.strip('"')
-                size = int(size)
-                sectors += size
-                extents.append([access_type, size, extent_type, filename])
+                # Extent descriptors can have up to seven values according to libvmdk documentation.
+                parts = line.split(" ", maxsplit=6)
+
+                if len(parts) < 3:
+                    # TODO: or should we raise a ValueError?
```

Comment:
Normally I'd say raise the error but I'm feeling adventurous today so log is fine.